### PR TITLE
[DA-18] [DA-30]: Add in-project dependencies, add plugins, add enforcers

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,25 +1,200 @@
...
-	<packaging>pom</packaging>
+    <groupId>org.jboss.da</groupId>
+    <artifactId>parent</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
 
-	<name>dependency-analyzer</name>
-	<url>https://github.com/project-ncl/dependecy-analysis</url>
+    <name>dependency-analyzer</name>
+    <url>https://github.com/project-ncl/dependecy-analysis</url>
```
Comment (jomrazek) [inline pom.xml]: You can add developers too

Comment (jbartece) [inline pom.xml]: Done

**Comment by jomrazek** [comment]
+1

**Comment by janinko** [comment]
Why not to update plugins to latest versions?

maven-compiler-plugin 3.3
maven-failsafe-plugin 2.18.1
maven-surefire-plugin 2.18.1
maven-ejb-plugin 2.5.1
maven-war-plugin 2.6
maven-dependency-plugin 2.10
maven-jar-plugin 2.6

**Comment by jbartece** [comment]
@janinko It is updated in the latest commit

**Comment by janinko** [comment]
Ok, then :+1:

**Pull request merged**